configure

```diff
@@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
 # Invoke python_config and set up symlinks to python includes
 (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
 
-## Set up Cuda-related environment settings
-
-while [ "$TF_NEED_CUDA" == "" ]; do
```

Comment:
+Kristina Chodorow <kchodorow@google.com> to answer that question.

On Thu, Jul 28, 2016 at 11:04 AM David Z. Chen <notifications@github.com>
wrote:

> In configure
> <https://github.com/tensorflow/tensorflow/pull/3269#discussion_r72587457>:
>
> > @@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
> >  # Invoke python_config and set up symlinks to python includes
> >  (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
> >
> > -## Set up Cuda-related environment settings
> > -
> > -while [ "$TF_NEED_CUDA" == "" ]; do
>
> Working on fixing these two issues.
>
> By the way, is there a reason why the fetch command doesn't create any
> Bazel output dir symlinks? Seems that it might be useful to create the
> bazel-<workspace-name> symlink at least so that we can look at the
> external repositories that were fetched.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/3269/files/d744bfcabe598d7940a0d3bccf2d1b20feb229fe#r72587457>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADjHfz8aFtMMQCkQ8dY6IX9ZyuOjLpRwks5qaHCogaJpZM4JJPRy>
> .
>